content/en/docs/zero-code/java/agent/sdk-configuration.md

```diff
@@ -0,0 +1,112 @@
+---
+title: SDK configuration
```

Comment:
Given that this page covers "Agent Configuration" **and** "SDK Configuration" as distinct sections, shouldn't this page be entitled just "Configuration" (as it was before), and the file name kept as `configuration.md`? And I don't think that you need an explicit qualifier of "General" before the "Configuration", it can be implicitly understood as general IMHO given, as @jeanbisutti pointed out, that much of this agent section is about config.

Reply:
Btw, if you choose to keep the new file name `sdk-configuration.md` then you'll need to **add an alias** for `configuration` (matching the previous filename).

Reply:
the idea was to keep a symetry with spring - @jeanbisutti wdyt?

Reply:
I'm feeling that we don't need to try to preserve such a correspondence here, but will wait for @jeanbisutti to chime in.

Reply:
> the idea was to keep a symmetry with spring - @jeanbisutti wdyt?

@zeitlinger "SDK configuration" and the symmetry with the Spring starter seem good to me

Reply:
@jeanbisutti - so you're ok with the "incoherence" that the title "SDK configuration" introduces by having page content that covers "Agent Configuration" and "SDK Configuration"?

Reply:
I think we'll come back to it one way or the other 😃

Reply:
Ok. For required followup actions, see https://github.com/open-telemetry/opentelemetry.io/pull/4811#pullrequestreview-2175221237. (Or you could revert the file and title name change and be done with it 😉)

Reply:
>so you're ok with the "incoherence" that the title "SDK configuration" introduces by having page content that covers "Agent Configuration" and "SDK Configuration"?

I had not noticed it  😱 The "SDK configuration" page is misleading for the Java agent. It contains several unrelated parts that are shorter to be located in specific pages, and most of them not related to the SDK configuration.

 The SDK configuration content of the Java agent is much shorter than that of the OTel starter.

I would suggest to move the content of this new "SDK configuration" page to the "Getting started" page.

Another option may be to have "Another configuration" page. The first option is perhaps less odd for users.

Reply:
I'm OOO next week, so I'll let you refactor that page as you see fit.